google/ggrc-core
# GGRC-7912 [FE] Disable fields that are taken from Issue Tracker when User is linking Issue to an existing ticket

# Issue description

Disable Status, Admins, Primary Contacts, Secondary Contacts fields that are taken from Issue Tracker when User is linking Issue to an existing ticket.

# Steps to test the changes

1. Switch on Ticket tracker.
2. Open an Issue with already linked ticket.
  a. Status, Admins, Primary Contacts, Secondary Contacts fields should be disabled. Tooltips should be displayed only for these fields.
  b. Turn off Ticket tracker. Status, Admins, Primary Contacts, Secondary Contacts fields should be enabled.
3. Open an Issue without linked ticket.
  a. Turn on Ticket tracker. Status, Admins, Primary Contacts, Secondary Contacts fields should be enabled.
  b. Click on 'Link to existing ticket' button. Status, Admins, Primary Contacts, Secondary Contacts fields should be disabled.
  c. Click on 'Generate new ticket' button. Status, Admins, Primary Contacts, Secondary Contacts fields should be enabled.
3. It is necessary to check that nothing is broken on the Assessments.


# Solution description

Add method in IssueTracker mixin which check if Issue is already linked or will be linked to an existing ticket. And add `issueTrackerDependent` property for people groups on which it is defined it is necessary to disable the field or not.

# Sanity checklist




# PR Review checklist
